inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/greenplum/GreenplumJdbcUtils.java

```diff
@@ -0,0 +1,275 @@
+     * @return true if column exist in the table, otherwise false
+     * @throws Exception on check column exist error
+     */
+    public static boolean checkColumnExist(final Connection conn, final String schemaName, final String tableName,
+            final String column)
+            throws Exception {
+        boolean result = false;
+        final String checkColumnSql = GreenplumSqlBuilder.getCheckColumn(schemaName, tableName, column);
+        try (Statement statement = conn.createStatement();
+                ResultSet resultSet = statement.executeQuery(checkColumnSql)) {
+            if (Objects.nonNull(resultSet) && resultSet.next()) {
+                int count = resultSet.getInt(1);
+                if (count > 0) {
+                    result = true;
+                }
+            }
+        }
+        LOG.info("check column exist for table={}, column={}, result={}", tableName, column, result);
+        return result;
+    }
+
+    /**
+     * Check whether the schema exists.
+     *
+     * @param conn JDBC Connection  {@link Connection}
+     * @param schemaName Greenplum schema name
+     * @return true if schema exist in the table, otherwise false
+     * @throws Exception on check column exist error
+     */
+    public static boolean checkSchemaExist(final Connection conn, final String schemaName) throws Exception {
+        boolean result = false;
+        if ("public".equals(schemaName)) {
+            result = true;
+        } else {
+            final String checkColumnSql = GreenplumSqlBuilder.getCheckSchema(schemaName);
+            try (Statement statement = conn.createStatement();
+                    ResultSet resultSet = statement.executeQuery(checkColumnSql)) {
+                if (Objects.nonNull(resultSet) && resultSet.next()) {
+                    int count = resultSet.getInt(1);
+                    if (count > 0) {
+                        result = true;
+                    }
+                }
+            }
+        }
+        LOG.info("check schema exist for schema={}, result={}", schemaName, result);
+        return result;
+    }
+
+    /**
+     * Query all columns of the tableName.
+     *
+     * @param conn JDBC Connection  {@link Connection}
+     * @param schemaName Greenplum schema name
+     * @param tableName Greenplum table name
+     * @return {@link List}
+     * @throws Exception on get columns error
+     */
+    public static List<GreenplumColumnInfo> getColumns(final Connection conn, final String schemaName,
+            final String tableName) throws Exception {
+        final List<GreenplumColumnInfo> columnList = new ArrayList<>();
+        final String querySql = GreenplumSqlBuilder.buildDescTableSql(schemaName, tableName);
+
+        try (Statement stmt = conn.createStatement();
+                ResultSet rs = stmt.executeQuery(querySql)) {
+            while (rs.next()) {
+                columnList.add(
+                        new GreenplumColumnInfo(
```

Comment:
Can those parameters be merged into one line?

Reply:
OK